### ISSUE TITLE
lepiej żeby ta funkcja zwróciła np. bool

### ISSUE THREAD

**Issue by bogkra** [bogkra/persons#10]
https://github.com/bogkra/persons/blob/9e5705fe1b7699450be4dbbd15eddf242c38c62a/persons.cpp#L80

**Comment by ziobron** [comment]
Tak, ogolnie bardzo dobrze jest jak funkcja coś zwraca. Często "studenckie funkcje" zwracają void, a informacja zwrotna jest przekazywana na wyjście za pomocą cout :-)
